NEWS.md

```diff
@@ -3,6 +3,9 @@
 * Added [Catch](https://github.com/philsquared/Catch) for unit testing of C++ code.
   See `?use_catch()` for more details. (@kevinushey)
 
+* `expect_identical()` and `is_identical_to()` now use `compare()` for more
```

Comment:
FWIW I think it's safer to always put new NEWS items at the top, so they're anchored at a fixed position.